sdk/analytics/src/main/java/com/klaviyo/analytics/DeviceProperties.kt

```diff
@@ -0,0 +1,82 @@
+package com.klaviyo.analytics
+
+import android.content.pm.PackageInfo
+import android.os.Build
+import com.klaviyo.core.BuildConfig
+import com.klaviyo.core.Registry
+import com.klaviyo.core.config.getPackageInfoCompat
+
+internal object DeviceProperties {
+
+    val manufacturer: String by lazy {
+        Build.BRAND
+    }
+
+    val model: String by lazy {
+        Build.MODEL
+    }
+
+    val platform: String by lazy {
+        "Android"
+    }
+
+    val osVersion: String by lazy {
+        Build.VERSION.SDK_INT.toString()
+    }
+
+    val appVersion: String by lazy {
+        packageInfo.versionName
+    }
+
+    val appVersionCode: String by lazy {
```

Comment:
many of these are obvious but some like this one may not be. Would it make sense to add some inline comments?